packages/lesswrong/components/ea-forum/EAHomeRightHandSide.tsx

```diff
@@ -30,17 +29,16 @@ import { useRecentOpportunities } from '../hooks/useRecentOpportunities';
 const styles = (theme: ThemeType): JssStyles => ({
   root: {
     paddingLeft: 40,
-    paddingRight: 32,
-    borderLeft: theme.palette.border.faint,
-    marginTop: 30,
+    paddingRight: 50,
+    marginTop: 10,
     marginLeft: 50,
     '@media(max-width: 1370px)': {
       display: 'none'
     }
```

Comment:
There's a small width window where the show/hide toggle is off the side of the screen, which also adds a horizontal scroll to the page:
<img width="1246" alt="Screen Shot 2023-09-27 at 6 06 29 PM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/42a19e20-0110-4eb5-8a31-7020db556c1c">

Basically since we got rid of the `borderLeft`, we no longer need the associated `paddingLeft`. Then you can prob fix the issue by fiddling with the `marginLeft` here and the `maxWidth` of various sections of the RHS.